.gitlab-ci.yml

```diff
@@ -27,13 +27,18 @@ before_script:
 vagrant_up:
   stage: vagrant_up
   script:
+    - vagrant box update
     - vagrant up
   when: always
 
 vagrant_tests:
   stage: vagrant_tests
   script:
     - vagrant ci test
+  coverage: '/ Coverage summary .+/'
```

Comment:
Does this run currently? I don't see no steenkeeng badgers

Reply:
The builds are set up but not integrated (still looking into the range of options with Gio):

https://gitlab.com/GPII/gpii-handlebars

The latest build did have a failure, I'll make sure it runs successfully once I finish with the rest of the feedback.